modules/admin_manual/pages/configuration/server/caching_configuration.adoc

```diff
@@ -47,6 +47,31 @@ The cache directory defaults to `data/$user/cache` where `$user` is the
 current user. You may use the `'cache_path'` directive in `config.php`
 (See config_sample_php_parameters) to select a different location.
 
+== Quick Configuration Example
+
+Install APCu and Redis:
+
+[source,console]
+----
+apt install php-apcu
+apt install redis-server php-redis
```

Comment:
These two lines could be combined into `apt install php-apcu redis-server php-redis`

Reply:
or we keep them separated. If one fails you know what failed. If the combined line fails you are not sure if it failed bacause of apcu or redis sever or php-redis.

Reply:
Afaik the output of `apt install` shows which packages are not able to be installed and why, such as a dependent package is not installable or held back.

Reply:
-.- okay, will change. faster than having this discussion.